rasa_core/training/interactive.py

```diff
@@ -148,18 +150,40 @@ def retrieve_tracker(endpoint, sender_id, verbosity=EventVerbosity.ALL):
     return _response_as_json(r)
 
 
-def send_action(endpoint, sender_id, action_name):
+def send_action(endpoint, sender_id, action_name, is_new=False):
```

Comment:
the developer could have implemented this new action and restarted the action server, in which case it would just work, even if the action wasn't in the domain yet.